atomic_cms.gemspec

```diff
@@ -7,6 +7,7 @@ Gem::Specification.new do |s|
   s.authors     = ['Don Humphreys']
```

Comment:
The long and short is, yes he does and I don't want to update this until I understand the ramifications of publishing the gem.